content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -68,6 +68,7 @@ spec:
     stdin: true
     tty: true
     resources:
+      # Note: If only limits are specified, kubelet will deduce requests from those limits and set them to be the same as the limits.
```

Comment:
How about adding this as a separate paragraph after the YAML?

Reply:
Do you mean to add it as Info alert? That will do as well

Reply:
Yup. Inline comments are usually used for something users easily get trapped or something you want to highlight. It may not be a good idea to introduce inline comments for some general concepts. After all, not all users need this reminder. Some users may feel a need for a reminder elsewhere.

Reply:
updated with note instead of yaml comment and placed it after output section